docs/slick-theme.css

```diff
@@ -4,6 +4,22 @@
 {
     background: #fff url('./ajax-loader.gif') center center no-repeat;
 }
```

Comment:
This pr includes only the dior accessibility changes, all tests pass and the carousel has been confirmed to be regression-free.

Another branch is coming based on this branch, that includes additional accessibility focus-related changes.